src/arrays/constructors.jl

```diff
@@ -16,4 +16,16 @@
     one_at_ind!(arr, i) = setindex!(arr, one(eltype(arr)), i)
     single_coeff(i, lens...) = one_at_ind!(zeros(lens), i)
 
-export statevec
\ No newline at end of file
+function coherentstatevec(n::Int,alpha::Number;method="direct")
+    s = rand(n)
```

Comment:
Just as an aside, why are you not using `zeros(n)` here? Calling `rand` will actually mean unnecessary computations. For small `n` this is fine, but for larger `n` you will notice the difference.